detectron2/modeling/backbone/resnet.py

```diff
@@ -608,6 +608,11 @@ def build_resnet_backbone(cfg, input_shape):
 
     stages = []
 
+    rem_feat = []
+    if "stem" in out_features:
+        out_features.remove("stem")
+        rem_feat.append("stem")
+
     # Avoid creating variables without gradients
     # It consumes extra memory and may cause allreduce to fail
     out_stage_idx = [{"res2": 2, "res3": 3, "res4": 4, "res5": 5}[f] for f in out_features]
```

Comment:
this minimal change 
```suggestion
    out_stage_idx = [{"res2": 2, "res3": 3, "res4": 4, "res5": 5}[f] for f in out_features if f != "stem"]
```
is probably enough

Reply:
Agreed. On a related note, should such a check be added for `linear` as well?

Reply:
I guess there's no need for it since you won't have that layer when you're using resnet as a backbone.